docs/cardinal/game/configuration/evm.mdx

```diff
@@ -0,0 +1,104 @@
+---
+title: 'EVM'
+description: 'This section configures the Ethereum Virtual Machine (EVM) settings, including chain IDs, mnemonics, and faucet configurations.'
+---
+
+```
+[evm]
+CHAIN_ID = string
+CHAIN_KEY_MNEMONIC = string
+DA_AUTH_TOKEN = string
+DA_BASE_URL = string
+DA_NAMESPACE_ID = string
+FAUCET_ADDRESS = string
+FAUCET_AMOUNT = string
+FAUCET_ENABLED = boolean
+```
+
+### CHAIN_ID
+
+A unique identifier representing the blockchain network. This is critical for identifying the correct chain to which the EVM is connected.
+
+**Importance** : Must be set to the correct chain ID to ensure proper blockchain communication.
+
+**Example**
+```
+CHAIN_ID = 'world-420'
+```
+
+### CHAIN_KEY_MNEMONIC
+
+Mnemonic phrases, also known as seed phrases or recovery phrases, used for generating cryptographic keys for your blockchain wallet.
+
+**Importance** : These should be stored securely, as they are used to recover the wallet.
+
+**Example**
+```
+CHAIN_KEY_MNEMONIC = 'enact adjust liberty squirrel bulk ticket invest tissue antique window thank slam unknown fury script among bread social switch glide wool clog flag enroll'
+```
+
+### DA_AUTH_TOKEN
+
+An authentication token obtained from the Celestia client, which is used to interact with Celestia’s Data Availability layer.
+
+**Importance** : Required for secure communication with the Celestia client.
+
+**Example**
+```
+DA_AUTH_TOKEN = 'ywi27664820'
+```
+
+### DA_BASE_URL
```

Comment:
Same as above. For all data availability (DA) layer-related configs (only celestia atm), we should also document the instructions to get these values.